opennlp-tools/src/test/java/opennlp/tools/ml/maxent/URLInputSreamFactory.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package opennlp.tools.ml.maxent;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+
+import opennlp.tools.util.InputStreamFactory;
+
+public class URLInputSreamFactory implements InputStreamFactory {
```

Comment:
Would this ever have any use outside of the test scope? Just asking because when I saw it I thought "Hey, this looks useful." But I really don't have any example use-cases in mind. I may be rambling.

It is missing a `t` in URLInputS**_t_**reamFactory.

Reply:
If the group wants, we can move it out of test.  I use this often, so I pulled it out of my local classes and dumped it in test.

Reply:
spelling fixed